.github/actions/oasis-core-version/action.yml

```diff
@@ -0,0 +1,14 @@
+name: Determine Oasis Core Version
```

Comment:
Minor (feel free to ignore): Should this be renamed to "Determine Oasis Core Commit Reference" or something in those lines? (also in other places)

Reply:
Thanks, looks good now!